src/Tribe/Admin/Notices.php

```diff
@@ -237,4 +238,44 @@ public function maybe_display_plus_commerce_notice() {
 			tribe_notice( "event-tickets-plus-missing-{$provider}-support", $message, 'dismiss=1&type=warning' );
 		}
 	}
+
+	/**
+	 * Display notices for unsupported currencies.
+	 *
```

Comment:
We need to put `@since TBD` every time we're adding a new method to our codebase. Then, when we release the version and the plugin is prepared by the release engineer, the TBD placeholder will be replaced with the version number.